config.js

```diff
@@ -19,7 +23,8 @@ exports.updateFromArgv = function(argv) {
     const debugStatus = argv.debug ? "DEBUG: ON\n" : "";
     console.log(`\ngtp2ogs version 6.0.1\n--------------------\n- For changelog or latest devel updates, please visit https://github.com/online-go/gtp2ogs/tree/devel\n${debugStatus}`);
 
-    // B - test unsupported argv
+    // B - throw errors
+    // all errors should be tested and thrown before we start exporting
```

Comment:
I'm not quite sure what this comment means. It's better to describe what we're going to do here, or not put a comment at all.

Reply:
this comment is related to pr #362 , which i dont think the idea is too good, the idea was to avoid the escape of `""` minmaxranks that dont throw an error even though they are invalid (a much simpler fix would have been to just do `!== undefined` instead of `if (argvRank)` in processRankExport, but i thought to generalize it (even though again this idea seems quite bad in the end, i may revert it to devel and close #362 , what do you think?)